src/Data/Managers/V2/GETTERS/GetInstanceWithRelations/useGetInstanceWithRelations.ts

```diff
@@ -29,15 +29,15 @@ interface GetInstanceWithRelationsHook {
  * React Query hook to fetch an instance with its related instances from the API. The related instances are all instances connected with given instance by inter-service relation, both, as a parent and as a child.
  * @param {string} id - The ID of the instance to fetch.
  * @param {string} environment - The environment in which we are looking for instances.
+ *  * @param {boolean} [includesReferencedBy] - A flag indicating if we should fetch instances that reference our main instance - defaults to false.
```

Comment:
Update docstrings too =) you still flagged it as optional, and have double asterix